src/main/java/com/example/Restaurant_Management_System_REST_API/service/ReservationService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.Restaurant_Management_System_REST_API.service;
+
+import com.example.Restaurant_Management_System_REST_API.model.entity.Reservation;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ReservationService implements GenericBasicCrudOperations<Reservation, Reservation, Long> {
+    @Override
+    public Reservation create(Reservation object) {
+        return null;
+    }
+
+    @Override
+    public Reservation read(Long aLong) {
+        return null;
+    }
+
+    @Override
+    public List<?> readAll() {
+        return null;
+    }
+
+    @Override
+    public Reservation update(Reservation object) {
+        return null;
+    }
+
+    @Override
+    public void delete(Reservation object) {
+
+    }
```

Comment:
I won't accept anything like that. We talked about it, Bartek. Unimplemented methods are not meant to be on the PRs due to the fact that when those will get on master it will be harder to catch any errors! This is unacceptable.

Reply:
but I will fulfill these methods in the next step, what is wrong with it? These services implements generic service therefore those methods must be overrided. This is a branch named `app_structure` If I finish these methods here it will no longer be a `app_structure` but the mixture of I dont know what - app structure and partially ready methods in services.

Reply:
The problem is that it'll land on master with these methods returning null. It's unacceptable, as I said previously.